### DISCUSSION TITLE
Allow elytra to be click equipped when allow-hot-swapping is enabled.

### DISCUSSION THREAD

**Discussion by RealTriassic** [PurpurMC/Purpur#466] [__Completed__] [answered]
I believe that elytra could be considered an armour piece, so therefore, I think you should be able to hot swap is out with your chestplate, thanks.

**Comment by BillyGalbreath** [comment, marked as answer]
Added in build 1245 https://github.com/pl3xgaming/Purpur/commit/b70ac0675f959785b2bfacab8647774c9fa0aa6b
